src/main/steps/common/support-you-need-during-case/summary/handler.ts

```diff
@@ -107,7 +113,16 @@ function processDocSupportData(userCase: Partial<CaseWithId>) {
   }
 }
 
-function reasonableAdjustment(userCase: Partial<CaseWithId>, urls: object) {
+function reasonableAdjustment(userCase: CaseWithId, urls: object, userId: string) {
+  const partyType = getCasePartyType(userCase, userId);
+  if (partyType === PartyType.RESPONDENT) {
+    respondentUrls(urls, userCase);
+  } else if (partyType === PartyType.APPLICANT) {
+    applicantUrls(urls, userCase);
+  }
+}
+
+function respondentUrls(urls: object, userCase: CaseWithId) {
   if (userCase.reasonableAdjustments?.includes(ReasonableAdjustments.DOCUMENTS_SUPPORT)) {
```

Comment:
Can u pls get rid of respondentUrls & applicantUrls methods and split each if condition as a es6 methods and then call it inside reasonableAdjustment function. Have the if conditions inside reasonableAdjustment method. So each individual method will hold a ternary operator for identifying the partyType.

Reply:
Changed if conditions to arrow methods as discussed